modules/ztp-pre-staging-partitioning.adoc

```diff
@@ -0,0 +1,210 @@
+
+.Procedure
+
+. Verify that the disk is cleared:
++
+[source,terminal]
+----
+# lsblk
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME    MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
+loop0     7:0    0  93.8G  0 loop /run/ephemeral
+loop1     7:1    0 897.3M  1 loop /sysroot
+sr0      11:0    1   999M  0 rom  /run/media/iso
+nvme0n1 259:1    0   1.5T  0 disk
+----
+
+. Erase any file system, RAID or partition table signatures from the device:
++
+[source,terminal]
+----
+# wipefs -a /dev/nvme0n1
+----
+
++
+.Example output
+[source,terminal]
+----
+/dev/nvme0n1: 8 bytes were erased at offset 0x00000200 (gpt): 45 46 49 20 50 41 52 54
+/dev/nvme0n1: 8 bytes were erased at offset 0x1749a955e00 (gpt): 45 46 49 20 50 41 52 54
+/dev/nvme0n1: 2 bytes were erased at offset 0x000001fe (PMBR): 55 aa
+----
+
+[IMPORTANT]
+====
+The tool fails if the disk is not empty because it uses partition number 1 of the device for precaching the artifacts.
+====
+
+[id="ztp-create-partition_{context}"]
+== Creating the partition
+
+Once the device is ready, you create a single partition and a GPT partition table.
+The partition is automatically labelled as `data` and created at the end of the device.
+Otherwise, the partition will be overridden by the `coreos-installer`.
+
+[IMPORTANT]
+====
+The `coreos-installer` requires the partition to be created at the end of the device and to be labelled as `data`. Both requirements are necessary to save the partition when writing the {op-system} image to the disk.
+====
+
+.Prerequisites
+
+* The container must run as `privileged` due to formatting host devices.
+* You have to mount the `/dev` folder so that the process can be executed inside the container.
+
+.Procedure
+
+In the following example, the size of the partition is 250 GB due to precaching Day 2 Operators for the DU profile.
+
+. Run the container as `privileged` and partition the disk:
++
+[source,terminal]
+----
+# podman run -v /dev:/dev --privileged \
+--rm quay.io/openshift-kni/telco-ran-tools:latest -- \
+factory-precaching-cli partition \ <1>
+-d /dev/nvme0n1 \ <2>
+-s 250 <3>
+----
+<1> Specifies the partitioning function of the {factory-prestaging-tool}.
+<2> Defines the root directory on the disk.
+<3> Defines the size of the disk in GB.
+
+. Check the storage information:
++
+[source,terminal]
+----
+# lsblk
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
+loop0         7:0    0  93.8G  0 loop /run/ephemeral
+loop1         7:1    0 897.3M  1 loop /sysroot
+sr0          11:0    1   999M  0 rom  /run/media/iso
+nvme0n1     259:1    0   1.5T  0 disk
+└─nvme0n1p1 259:3    0   250G  0 part
+----
+
+.Verification
+
+You must verify that the following requirements are met:
+
+* The device has a GPT partition table
+* The partition uses the latest sectors of the device.
+* The partition is correctly labeled as `data`.
+
+. Query the disk status to verify that the disk is partitioned as expected:
```

Comment:
If you remove the step bullet `.` and get rid of the `+` lsit continuation `+` symbols, this might be formatted better.

```suggestion
Query the disk status to verify that the disk is partitioned as expected:
```